packages/reports/app/styles/navi-reports/components/visualization-selector.less

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+.visualization-selector {
+  background-color: @denali-brand-100;
+  display: flex;
+  flex-direction: column;
+  padding: 15px 0;
```

Comment:
will it look better without the top padding?
![Screen Shot 2020-01-08 at 11](https://user-images.githubusercontent.com/13946669/72011241-43675d80-320e-11ea-9a34-d71fb7eceb7a.png)

Reply:
I think it looks okay that way too, but I was just following the initial mock in the Jira ticket.